Vision/README.md

```diff
@@ -83,12 +83,14 @@ serving the Vision for the Web.
 	openly developed with consensus of industry and key stakeholders.
 
 # Mission of the W3C
-The fundamental purpose of the W3C is to provide an open forum, 
-where diverse voices from around the world and industries come together, 
-incubate, and build consensus for global standards for Web technologies, 
-including organizations of all sizes 
-(from single-person companies to multi-nationals) 
-and considering public feedback.
+The fundamental purpose of the W3C is to provide an open forum 
+where diverse voices from around the world
+work together to build consensus
+on global standards for Web technologies. 
+This forum includes organizations of all sizes
+(from single-person companies to multi-nationals),
+representatives of industries as well as individuals,
+and considers public feedback.
```

Comment:
```suggestion
representatives of industries as well as individuals.
The W3C also requests, and takes into active consideration, 
feedback on its work from people and organisations who are not members.
```

Reply:
This seems way too long and convoluted.

Reply:
Agreed. On the other hand "considers public feedback" seems too bland. It doesn't make clear that there is a conscious goal to get it, and ensure that it can influence the outcomes.

I was also thinking about how to note that feedback is generally considered from a "technical" perspective, including that e.g. the political or social impact of something is part of that perspective (hence things like privacy, accessibility, ...) but didn't get there yet - and it might fit better somewhere else in the document.

Reply:
Possible rephrasing?

```
The fundamental purpose of the W3C is to provide an open forum 
where diverse voices from around the world
and from different industries
work together to build consensus
on global standards for Web technologies. 
This forum welcomes organizations of all sizes
(from single-person companies to multi-nationals),
and takes into active consideration
feedback from the general public.
```

This shifts the various industries clause up alongside "around the world", which allows the last sentence to have a cleaner structure.

Reply:
Taken update, with a slight tweak.